src/modules/Modules.cpp

```diff
@@ -82,9 +82,11 @@ void setupModules()
 
         storeForwardModule = new StoreForwardModule();
 
-        new RangeTestModule();
 #elif defined(ARCH_NRF52)
         externalNotificationModule = new ExternalNotificationModule();
+#endif
+#if defined(ARCH_ESP32) || defined(ARCH_NRF52)
+        new RangeTestModule();
```

Comment:
Not scope, but you can shove the externalNotification init there as well for clarity, removing the dual definition and elif above. :-)

Reply:
Good call. We've got enough intense macro gaurding in there